src/app/shared/components/number-stepper/style.scss

```diff
@@ -43,5 +43,5 @@ input[type='number']::-webkit-outer-spin-button {
 }
 
 .mat-form-field {
-  min-width: 150px;
+  min-width: 130px;
```

Comment:
Do not change this globally as it will update our whole UI and many other places. You can create a custom class that when applied changes the width.